### ISSUE TITLE
Can't upload 11MB zip file to pgxn manager

### ISSUE THREAD

**Issue by sagneta** [pgxn/pgxn-manager#51]
I get the following error when I attempt to do so:

413 Request Entity Too Large

The reason my zip is this large is because it also contains a Visual Studio project file to build the extension on a Windows box. Silly Windows.....

**Comment by depesz** [comment]
I temporarily changed limit in the webserver, please try again and let me know.

**Comment by theory** [comment]
Yeah, there's no limit in the code, so @depesz's fix should address the issue.

**Comment by sagneta** [comment]
That worked! Thank you guys. I appreciate it. #

**Comment by beaud76** [comment]
I get the same error message "413 Request Entity Too Large" from nginx/1.6.2 when I try to upload a new version of the E-Maj extension. The file is only 2.4 Mb and I succeeded to upload a little bit larger file last September, and a similar size mid November.
(I tried from 2 different workstations, and also after having deleted the browser history and cache as suggested by some threads found on the web).
Has a max upload size limit being lowered ?
